src/plugins/Sharesite/webui/EditToadlet.java

```diff
@@ -250,21 +274,28 @@ public void handleMethodPOST(URI uri, HTTPRequest req, ToadletContext ctx) throw
 		// Perform action
 		if (req.isPartSet("saveBtn") || req.isPartSet("previewBtn")||req.isPartSet("preprocessBtn")) {
 			String name = req.getPartAsStringFailsafe("nameInput", 1000).trim(); // 1kB
+			String path = req.getPartAsStringFailsafe("pathInput", 1000).trim(); // 1kB
 			String desc = req.getPartAsStringFailsafe("descInput", 10000).trim(); // 10kB
 			String text = req.getPartAsStringFailsafe("textInput", 1000000000).trim(); // 1GB
```

Comment:
not my code :) (where is the "hand waving, I’m not touching this and hope there’s a reason"-emoji?)

Anyway: This is strange, but not problematic, since it would only be the user hurting him/herself (freenet should handle uploading 1GB of data).